packages/liveblocks-core/src/room.ts

```diff
@@ -1735,6 +1735,26 @@ function makeStateMachine<
 
             break;
           }
+
+          // Receiving a RejectedOps message in the client means that the server is no
+          // longer in sync with the client. Trying to synchronize the client again by
+          // rolling back particular Ops may be hard/impossible. It's fine to not try and
+          // accept the out-of-sync reality and throw an error. We look at this kind of bug
+          // as a developer-owned bug. In production, these errors are not expected to happen.
+          case ServerMsgCode.REJECT_STORAGE_OP: {
+            console.errorWithTitle(
+              "Storage op rejection error",
+              message.reason
+            );
+
+            if (process.env.NODE_ENV !== "production") {
+              throw new Error(
+                `Storage ops rejected by server: ${message.reason}`
```

Comment:
While working on the [error message screenshots](https://github.com/liveblocks/liveblocks/pull/647#discussion_r1095647226), I noticed the use of the word "op" here, which is a concept that most Liveblocks users will never see. Let's avoid it in public messages:

```suggestion
            console.errorWithTitle(
              "Storage mutation rejection error",
              message.reason
            );

            if (process.env.NODE_ENV !== "production") {
              throw new Error(
                `Storage mutation rejected by server: ${message.reason}`
```